utils/test-manager.js

```diff
@@ -65,40 +74,58 @@ class TestManager {
 
   async getNonceForRelay() {
     const block = await this.provider.getBlockNumber();
-    const timestamp = (new Date()).getTime();
-    return `0x${ethers.utils.hexZeroPad(ethers.utils.hexlify(block), 16)
-      .slice(2)}${ethers.utils.hexZeroPad(ethers.utils.hexlify(timestamp), 16).slice(2)}`;
+    const timestamp = new Date().getTime();
+    return `0x${ethers.utils
+      .hexZeroPad(ethers.utils.hexlify(block), 16)
+      .slice(2)}${ethers.utils
+      .hexZeroPad(ethers.utils.hexlify(timestamp), 16)
+      .slice(2)}`;
```

Comment:
There's a lot of reformatting that has gone in this file, maybe to do with prettier settings in editor or something like that. Please reformat everything back.

Reply:
Is that still the case? `eslint .` does not show me any warning or error for this file.

Reply:
Reformatted.